aclocal.m4

```diff
@@ -1,6 +1,6 @@
-# generated automatically by aclocal 1.16.3 -*- Autoconf -*-
```

Comment:
We need to use the same versions when regenerating these. I believe the version mismatch is why the generated configure script deletes some unrelated runstatedir lines as well.

https://devguide.python.org/setup/#regenerate-configure